app/questionnaire/placeholder_parser.py

```diff
@@ -80,7 +80,12 @@ def _resolve_answer_value(self, value_source):
                 self._lookup_answer(each_identifier, list_item_id)
                 for each_identifier in value_source["identifier"]
             ]
-        return self._lookup_answer(value_source["identifier"], list_item_id)
+        answer = self._lookup_answer(value_source["identifier"], list_item_id)
+        return (
+            answer.get(value_source["selector"])
+            if "selector" in value_source
+            else answer
+        )
```

Comment:
It's not been introduced by this change, but this highlights that the values in `dict` answers aren't being escaped. We can either address it in this pull request, or raise a card for it?

Reply:
You can test this by adding html into the address line 1 answer and seeing it rendered on the confirmation page.

Reply:
If we would usually store HTML and escaped on playback, then it realistically should be this card. The only reason I therefore would suggest a different story would be because of the testing deadline for DAC tomorrow. It wouldn't matter to them